test/falco_tests.yaml

```diff
@@ -292,20 +295,10 @@ trace_files: !mux
       - rules/invalid_array_item_not_object.yaml
     trace_file: trace_files/cat_write.scap
 
-  invalid_unexpected object:
```

Comment:
Is this deletion intended?

Reply:
I saw that you deleted `test/rules/invalid_unexpected_object.yaml`. Is this test not needed anymore?

Reply:
Yes, this test was removed as the rule loading is now tolerant of unknown top level objects. This will allow defining new top level objects (something analogous to rule/exception/macro/list) without interfering with older falco instances that might not know about them.

Reply:
Ok, that sounds like something to mention in the release notes :)

Because users may find ways to use this feature to annotate Falco rules ;)